usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Point.java

```diff
@@ -0,0 +1,61 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+/**
+ * @author Marc Simó
+ */
+public class Point {
+    private Double latitude;
+    private Double longitude;
+
+    /**
+     * Point constructor.
+     */
+    public Point() { }
+
+    /**
+     * Point constructor specifying coordinates.
+     * @param latitude Point latitude
+     * @param longitud Point longitude
+     */
+    public Point(Double latitude, Double longitud) {
+        this.latitude = latitude;
+        this.longitude = longitud;
+    }
+
+    public Double getLatitude() {
+        return latitude;
+    }
+
+    public void setLatitude(Double latitude) {
+        this.latitude = latitude;
+    }
+
+    public Double getLongitude() {
+        return longitude;
+    }
+
+    public void setLongitude(Double longitude) {
+        this.longitude = longitude;
+    }
+
+    @NonNull
+    @Override
+    public String toString() {
+        return "{" +
+            "latitude=" + latitude +
+            ", longitude=" + longitude +
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['+' should be on a new line.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5504923)